src/controller/user.controller.js

```diff
@@ -0,0 +1,95 @@
+const userService = require('../services/user.service');
+
+const get = async (req, res) => {
+  const users = await userService.getAll();
+
+  if (!users) {
+    res.send(users);
+    res.sendStatus(404);
```

Comment:
You are sending the users in the response and then immediately sending a 404 status code. This would result in an error because headers cannot be sent after the response is sent. You should send a 404 status code if no users are found.